### DISCUSSION TITLE
For a regular user, when should I use packages from store and when from winget?

### DISCUSSION THREAD

**Discussion by totkeks** [microsoft/winget-cli#2780] [General] [answered]
I have looked for this question and didn't find any existing discussion. Also I'm not a sysadmin managing multiple machines, just a regular PC user.

I have been wondering for a while now, what is the goal for winget and the store (as a source in winget). 

Should I install packages from the store or winget package sources? How can I know the difference?

For example Powershell, windows terminal, power toys, but of course many more apps that don't happen to be incidentally made by Microsoft.

Is there kind of a general consensus? Or is it a "it depends"? And if yes, what does it depend on?

Thanks for reading. I would really love to get some thoughts from other people about this.

**Comment by Trenly** [comment, marked as answer]
The winget source is compiled from the [community repository](/microsoft/winget-pkgs) and may contain packages that aren't available in the microsoft store. The msstore source is the same as the [Microsoft Store](https://apps.microsoft.com/store/apps), meaning that all the packages in the msstore source are from verified publishers. 

Really, it depends on what your personal preferences are. If you want only "official" apps - use the msstore source. If you're okay with going to a browser and downloading an executable file to install an application - the winget source is great.

**Reply by totkeks** [reply]
Sorry for the late reply. I think I wasn't precise enough in my initial description. What I was trying to ask is, if there is a package in both the community repository and the MS store - like PowerShell or VS Code - which package source is the recommended one? If there even is such a thing. For me such a recommendation would help and maybe for other users too.

What I'm mostly trying to understand is, what Microsoft's long-term strategy is behind both sources.

**Reply by Masamune3210** [reply]
There really isnt a recommendation, they both do the same thing, just different ways. The only real benefit to using the store package is updates to the program being automatically handled by the Store in some cases. Legacy MSI and MSIX both exist to solve the same issue, getting the program onto your computer in a usable and trackable fashion.
